SingularityS3Base/src/main/java/com/hubspot/singularity/s3/base/config/SingularityS3Configuration.java

```diff
@@ -67,6 +67,23 @@
   @JsonProperty
   private String localDownloadPath = "/download";
 
+  @NotEmpty
+  @JsonProperty
+  private String gpgBinaryPath = "/usr/bin/gpg";
+
+  @NotEmpty
+  @Obfuscate
+  @JsonProperty
+  private String gpgKeyPassword;
+
+  @NotEmpty
```

Comment:
As these are NotEmpty does it imply gpg is now a "required" part of the Singularity S3 configuration?